### DISCUSSION TITLE
Advice on Polling limits when using the new Fleet API please

### DISCUSSION THREAD

**Discussion by abreingan** [alandtse/tesla#1001] [Q&A]
Till today I have been using TeslaMate to retrieve data and the Fleet API for commands (for solar charging), and this worked well, but TeslaMate now seems to be rate limited (with Tesla suggesting 5 minute poll).
I had therefore turned polling on the Tesla Custom Integration to once a day, assuming this might contribute to the 50 commands per day limit, as I have just been able to live under that limit.

I note that polling seems to be able to be turned on or off by automation and the force data update does what it says on the tin.

It might be valuable to others (it certainly would to me) to have anyone describe how they manage solar charging under the new regime. 
Does polling "cost" a command? 
Does force data update?

**Comment by jesperholstjh** [comment]
Short answer; They don't. Rate limiting is rate limiting.

Instead some are looking for a BLE solution.

Look here https://github.com/alandtse/tesla/issues/961#issuecomment-2150897886

**Comment by nickt444** [comment]
I dont think polling for data updates counts against the 50/day limit (for the simple reason that it still works after I start getting the dreaded 429 errors for exceeding the rate command limit). I changed from polling every 1 min to every 10 mins and changing charge rate according to excess solar capacity. It works ok, but still sometimes exceeds the cap in the afternoon (especially on cloudy days). I will probably switch to BLE in the near future
